packer/bastion/user_data.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+echo "***** Update *****"
+sudo yum update -y
+
+echo "***** Install pip *****"
+curl -O https://bootstrap.pypa.io/get-pip.py
+sudo python get-pip.py
+
+echo "***** Update awscli *****"
+sudo pip install --upgrade awscli
+
+echo "***** Install epel and figlet for setting up banner *****"
+sudo amazon-linux-extras install epel -y
+sudo yum install figlet -y
```

Comment:
Add in bastion motd then remove figlet

Reply:
This is also dependent on instance data $IP, $AVAILABILITY_ZONE and $INSTANCE_TYPE, so it needs to be in the user data.